src/sentry/integrations/msteams/webhook.py

```diff
@@ -180,3 +199,63 @@ def handle_member_removed(self, request):
 
         integration.delete()
         return self.respond(status=204)
+
+    def handle_action_submitted(self, request):
+        data = request.data
+        channel_data = data["channelData"]
+        team_id = channel_data["team"]["id"]
+        tenant_id = channel_data["tenant"]["id"]
+        group_id = data["value"]["groupId"]
+        user_id = data["from"]["id"]
+
+        try:
+            integration = Integration.objects.get(provider=self.provider, external_id=team_id)
+        except Integration.DoesNotExist:
+            logger.info("msteams.action.missing-integration", extra={"team_id": team_id})
+            return self.respond(status=404)
+
+        try:
+            group = Group.objects.select_related("project__organization").get(
+                project__in=Project.objects.filter(
+                    organization__in=integration.organizations.all()
+                ),
+                id=group_id,
+            )
+        except Group.DoesNotExist:
+            logger.info(
+                "msteams.action.invalid-issue",
+                extra={"team_id": team_id, "integration_id": integration.id},
+            )
+            return self.respond(status=404)
+
+        try:
+            idp = IdentityProvider.objects.get(type="msteams", external_id=team_id)
+        except IdentityProvider.DoesNotExist:
+            logger.info(
+                "msteams.action.invalid-team-id",
+                extra={
+                    "team_id": team_id,
+                    "integration_id": integration.id,
+                    "organization_id": group.organization.id,
+                },
+            )
+            return self.respond(status=404)
+
+        try:
+            identity = Identity.objects.select_related("user").get(idp=idp, external_id=user_id)
+        except Identity.DoesNotExist:
+            associate_url = build_linking_url(
+                integration, group.organization, user_id, team_id, tenant_id
+            )
+
+            client = MsTeamsClient(integration)
+
+            card = build_linking_card(associate_url)
+            user_conversation_id = client.get_user_conversation_id(user_id, tenant_id)
+            client.send_card(user_conversation_id, card)
+            return self.respond(status=201)
+
+        if False:
```

Comment:
@KKaempen Pretty sure this is not supposed to be here :)

Reply:
Is there some standard way to get flake8 to not complain about unused variables that I'm not aware about? Because this was my janky solution to that. :|